docs/docsite/rst/dev_guide/testing/sanity/future-import-boilerplate.rst

```diff
@@ -1,10 +1,8 @@
-Sanity Tests » boilerplate
```

Comment:
Keep the original `boilerplate` test docs even though the test is gone. That way links to the docs from old PRs will continue to function.

Reply:
Done.